CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ master (unreleased)
 * Support objects created with Object.create(null). fixes [#468](https://github.com/mozilla/nunjucks/issues/468)
 
 * Support ESNext iterators, using Array.from. Merge of
-  [#760](https://github.com/mozilla/nunjucks/pull/760)
+  [#1058](https://github.com/mozilla/nunjucks/pull/1058)
```

Comment:
Should I add: fixes #759?

Reply:
I'm planning on merging develop into master later today, and I'll have to add a few entries to the CHANGELOG myself, so I'll make sure to mention that ticket.

Reply:
Thanks